core/src/Kokkos_NumericTraits.hpp

```diff
@@ -135,20 +135,32 @@ template <> struct denorm_min_helper<float> { static constexpr float value = __F
 template <> struct denorm_min_helper<double> { static constexpr double value = __DBL_DENORM_MIN__; };
 template <> struct denorm_min_helper<long double> { static constexpr long double value = __LDBL_DENORM_MIN__; };
 #endif
+// GCC <10.3 is not able to evaluate T(1) / norm_min<T>::value at compile time when passing -frounding-math
+// https://godbolt.org/z/zj9svb1T7
+// Similar issue was reported on IBM Power without the compiler option
+#define KOKKOS_IMPL_WORKAROUND_CONSTANT_EXPRESSION_COMPILER_BUG
```

Comment:
Do we need to add an `#ifndef KOKKOS_IMPL_WORKAROUND_CONSTANT_EXPRESSION_COMPILER_BUG` above this line?  Not sure what the convention is in Kokkos for using that macro.

Reply:
No it is only meant for local use.  I might as well have commented the code below.